src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs,
+  selectedTabId,
+  setSelectedTabId,
+  onTabSelected }) => {
+  const handleClick = (tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tabs.find(tabsTab => tabsTab.id === tab.id));
+    }
+  };
+
+  const getIsActive = (tab) => {
+    if (tabs.find(tabsTab => tabsTab.id === selectedTabId) === undefined
+    && tab.id === tabs[0].id) {
+      return 'is-active';
+    }
```

Comment:
```suggestion

```

Reply:
there was 1 error, after this change we have 2 errors